benchmarks/ctia/entity/actor_bench.clj

```diff
@@ -0,0 +1,30 @@
+(ns ctia.entity.actor-bench
+  (:require [ctia.test-helpers
+             [benchmark :refer [cleanup-ctia!
+                                setup-ctia-es-store!]]
+             [core :as helpers :refer [delete post]]]
+            [ctim.examples.actors
+             :refer [new-actor-minimal
+                     new-actor-maximal]]
+            [perforate.core :refer :all]))
+
+(def small-actor new-actor-minimal)
+(def big-actor (dissoc new-actor-maximal :id))
+
+(defgoal create-actor "Create Actor"
+  :setup (fn [] [(setup-ctia-es-store!)])
+  :cleanup (fn [_] cleanup-ctia!))
+
+(defn play [fixture]
+  (let [{:keys [status parsed_body]}
+        (post "ctia/actor"
+              :body fixture
+              :headers {"Authorization" "45c1f5e3f05d0"})]
+    (if (= 201 status)
+      (prn "play-big: " status))))
```

Comment:
I think we should get rid of the complete block here. It will always generate a `(prn "play-big: 201")` and it takes a bit of time for IOs and as the only operation is a post, that time might not be totally marginal.